package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "frontend-burgernabije-besluitendatabank",
-  "version": "0.8.1",
+  "version": "0.8.2",
```

Comment:
Why was this bumped here? Bumping in branches isn't a good idea :smile: If you want an image to test before merging I would use the "feature" image builds.